csharp/src/Ice/Proxy.cs

```diff
@@ -463,7 +463,7 @@ async Task<IncomingResponseFrame> InvokeAsync()
                             {
                                 // 1.1 System exceptions
                                 lastException = systemException;
-                                if (reference.RouterInfo != null && one.Operation == "ice_add_proxy")
+                                if (reference.RouterInfo != null && one.Origin!.Value.Operation == "ice_add_proxy")
```

Comment:
Is this the main or only use-case for making the set on RemoteException.Origin public?

Reply:
What is the point of adding it to all remote exceptions if you cannot access this?

Reply:
I was talking about the set, not the get.

Reply:
let's make it internal